complete/application/src/test/java/com/example/multimodule/application/DemoApplicationTest.java

```diff
@@ -1,16 +1,15 @@
 package com.example.multimodule.application;
 
-import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
I thought we preferred AssertJ? Why replace this?

Reply:
It wasn't working. When I looked up how to replace it, I found the Hamcrest stuff. If you can show me how to get AssertJ to work with JUnit 5, I'd be fine with changing it.

Reply:
It should just work. The Boot team use AssertJ, so you could look at some of their tests I guess. E.g. https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot/src/test/java/org/springframework/boot/builder/SpringApplicationBuilderTests.java

Reply:
I got AssertJ to work. Thanks, Dave.